artiq/frontend/artiq_master.py

```diff
@@ -53,7 +53,9 @@ def get_argparser():
         "--experiment-subdir", default="",
         help=("path to the experiment folder from the repository root "
               "(default: '%(default)s')"))
-
+    group.add_argument(
+        "--log-filename", default=None, 
```

Comment:
There are already other "logging" options in the master and this is confusing. ``--log-submissions``?